en/2-deeds.md

```diff
@@ -3,17 +3,17 @@
 
 ## 2.1. Deed NFTs
 
-In the digital realm, Non-Fungible Tokens (NFTs) serve a similar purpose to deeds in the physical world. Like a deed that confirms ownership of tangible property, NFTs establish indisputable ownership of digital assets. This ownership is securely recorded on a blockchain, ensuring that each asset is unique and non-replicable.
+In the digital realm, Non-Fungible Tokens (NFTs) serve a similar purpose to deeds in the physical world. Like a deed confirming tangible property ownership, NFTs establish indisputable ownership of digital assets. This ownership is securely recorded on a blockchain, ensuring each asset is unique and non-replicable.
 
-Within the WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Holding a Deed empowers its owner to manage a Community Hub and makes them eligible for receiving Mintium rewards, a form of recognition and incentive within the WoM ecosystem (further details in section 2.6).
```

Comment:
In the context of WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Owning a Deed empowers individuals to manage a Community Hub and qualifies them for Mintium rewards—a form of recognition and incentive within the WoM ecosystem (refer to section 2.6 for additional details).